pkg/secrethub/secret_version.go

```diff
@@ -115,7 +117,13 @@ func (s secretVersionService) GetWithData(path string) (*api.SecretVersion, erro
 		return nil, errio.Error(err)
 	}
 
-	return s.get(secretPath, true)
+	secretVersion, err := s.get(secretPath, true)
+	if api.IsErrNotFound(err) {
```

Comment:
How does this impact the use of `api.IsErrNotFound`? Will the wrapped error that we now return change the output of that function? Where is it being used? Is that a problem?

Reply:
Opened a PR on the CLI to use `api.IsErrNotFound`:
https://github.com/secrethub/secrethub-cli/pull/252 
It should be merged first.

The current `api.IsErrNotFound` does not work for this new error. However I am not sure how to elegantly check if an error has been created with `ErrSecretNotFound`.

Reply:
If you make `ErrSecretNotFound` a struct you can use [`errors.As`](https://golang.org/pkg/errors/#As). See https://blog.golang.org/go1.13-errors for an example.

Reply:
Wouldn't that be inconsistent with regards to how we currently create and handle errors?
I think that for now we could (inelegantly) check the namespace and code of the error and add error wrapping to the `errio` package later, to simplify this check. What do you think, @SimonBarendse ?

Reply:
@Marton6 You're right that this is inconsistent with our current error checking. To get dynamic error messages however, we do have to deviate from our current error checking method. Your proposal to check for namespace and code also deviates from our current approach.

What might be important to know is that we want to get rid of the `errio` package (see also https://github.com/secrethub/secrethub-go/pull/172#discussion_r384629641), so using something different here can be the start of moving away from the `errio` package.